examples/fargate-serverless/main.tf

```diff
@@ -137,12 +131,15 @@ module "eks_blueprints_kubernetes_addons" {
 
   enable_aws_load_balancer_controller = true
   aws_load_balancer_controller_helm_config = {
-    values = [
-      <<-EOT
-      clusterName: ${module.eks_blueprints.eks_cluster_id}
-      region: ${local.region}
-      vpcId: ${module.vpc.vpc_id}
```

Comment:
we previously had to replicate the values file here in order to set the `vpcId` for running the AWS LBC on Fargate. now we can set just the additional values we need to provide to the AWS LBC addon